src/encode.js

```diff
@@ -0,0 +1,33 @@
+const { extractMethodObjectsFromABI, loadABI } = require('./utils')
+const Web3 = require('web3')
+
+module.exports = function(abiPath, methodCall, url) {
+  if (!methodCall) {
+    throw new Error('[encode] methodCall required')
+  }
+
+  const isValidMethod = /^(\w+)\((.*)\)$/
+  const method = isValidMethod.exec(methodCall)
```

Comment:
I think that this way of detecting if a method is valid, should be in the utils file, since it could be useful, can you move it? Thanks

Reply:
@mariano-aguero, extracted to a method that now returns an object with the `methodCall` parts. I did so because the returned information will remain the same if for any reason the way the method is validated changes.